Dockerfile

```diff
@@ -44,10 +44,8 @@ COPY --from=cilium-envoy / /
 COPY plugins/cilium-cni/cni-install.sh /cni-install.sh
 COPY plugins/cilium-cni/cni-uninstall.sh /cni-uninstall.sh
 COPY contrib/packaging/docker/init-container.sh /init-container.sh
-WORKDIR /root
+WORKDIR /home/cilium
 RUN groupadd -f cilium \
-	&& echo ". /etc/profile.d/bash_completion.sh" >> /root/.bashrc \
```

Comment:
Won't this cause the image to not have any bash comletion at all? Maybe we should move bash completion to `/home/cilium/.bashrc`?

Reply:
See commit message in 9a65f5d8e4ecf54bf3449eccdd9593d595c79461.

I have tested it and `/etc/bash_completion.d/cilium` is already in place.

I did some more testing just now, and found that `/etc/bash_completion.d/cilium` gets loaded in a login shell, but doesn't get loaded in non-login shell.

So this works:
```
$ docker run -ti cilium bash -l
root@37e9f10b473c:/home/cilium# cilium 
bpf         completion  debuginfo   fqdn        kvstore     metrics     node        prefilter   service     version     
cleanup     config      endpoint    identity    map         monitor     policy      preflight   status      
root@37e9f10b473c:/home/cilium#
```
But this doesn't:
```
$ docker run -ti cilium bash   
root@f22ad80350d1:/home/cilium# cilium 
cilium                   cilium-bugtool           cilium-health            cilium-map-migrate       
cilium-agent             cilium-envoy             cilium-health-responder  cilium-node-monitor      
root@f22ad80350d1:/home/cilium# cilium 
```

I thin the fix should be simple, I'll have a look now.

Reply:
By the way, the reason I wanted to get rid of these lines is to avoid references to `/root`.